packages/react-composites/tests/app/call/BaseApp.tsx

```diff
@@ -38,9 +38,26 @@ export function BaseApp(props: { queryArgs: QueryArgs; callAdapter?: CallAdapter
     ? () => alert('you are using a unsupported browser')
     : undefined;
 
+  const onPermissionsTroubleshootingClick = queryArgs.useCallReadiness
+    ? (permissionsState: { camera: PermissionState; microphone: PermissionState }) => {
```

Comment:
Please define a file-private free function. Currently this callback is recreated each time `BaseApp` is rendered.

Similar comments for all the callbacks below.